ic-agent/src/agent/mod.rs

```diff
@@ -428,6 +495,37 @@ impl Agent {
     }
 }
 
+fn initialize_bls() -> Result<(), AgentError> {
```

Comment:
This file is getting big, does it make sense to move this, `extract_der`, and the lookup functions to a different file?

Reply:
Good idea!